### PR TITLE
Openapi swagger UI

### PR DESCRIPTION
**Pull request by arkadius**

### CODE REVIEW

File: engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/SwaggerUiRoute.scala [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import akka.http.scaladsl.server.{Directives, Route}
+import com.typesafe.scalalogging.LazyLogging
+
+object SwaggerUiRoute extends Directives with LazyLogging {
+
+  val route: Route = {
+    path("swagger-ui") {
+      get {
+        encodeResponse {
+          respondWithHeader(`Cache-Control`(List(CacheDirectives.public, CacheDirectives.`max-age`(0)))) {
```
Comment (mproch) [inline engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/SwaggerUiRoute.scala:13]: let's have sth like:
```    
pathEndOrSingleSlash {
      getFromFile(dir + "/index.html")
    } ~ 
    getFromDirectory(dir)
```

**Review by mproch — COMMENTED** [review]
Added a small comment

File: engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/SwaggerUiRoute.scala (hunk 1 above)
Comment (arkadius) [reply engine/lite/request-response/runtime/src/main/scala/pl/touk/nussknacker/engine/requestresponse/SwaggerUiRoute.scala:13]: done

**Review by mproch — APPROVED** [review]
:+1: cool

**Pull request merged**